terraform/dev/main.tf

```diff
@@ -0,0 +1,77 @@
+###
```

Comment:
This Terraform configuration file for `dev` is almost identical to the `/staging/main.tf`, besides the environment suffixes in the RDS and S3 services. Keeping these configs different by environment is the recommended best practice from Terraform, and provides a true declarative interface for all your infrastructure, where little is implicit.